### PR TITLE
CLOUD-391 added hdp, removed single

### PR DESCRIPTION
**Pull request by matyix**

### CODE REVIEW

File: src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy [hunk 1]
```diff
@@ -266,7 +266,7 @@ class CloudbreakClient {
 
     def void addDefaultBlueprints() throws HttpResponseException {
         postBlueprint("multi-node-hdfs-yarn", "multi-node-hdfs-yarn", getResourceContent("blueprints/multi-node-hdfs-yarn"), false)
-        postBlueprint("single-node-hdfs-yarn", "single-node-hdfs-yarn", getResourceContent("blueprints/single-node-hdfs-yarn"), false)
+        postBlueprint("hdp-multinode-default", "hdp-multinode-default", getResourceContent("blueprints/hdp-multinode-default"), false)
         postBlueprint("lambda-architecture", "lambda-architecture", getResourceContent("blueprints/lambda-architecture"), false)
         postBlueprint("warmup", "warmup", getResourceContent("blueprints/warmup"), false)
```
Comment (doktoric) [inline src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy]: you removed the warmup

**Comment by matyix** [comment]
removed now ... thx

File: src/main/resources/blueprints/lambda-architecture [hunk 2]
```diff
@@ -77,7 +77,166 @@
           "name": "PIG"
```
Comment (doktoric) [inline src/main/resources/blueprints/lambda-architecture]: doublecheck this bp pls

**Comment by doktoric** [comment]
lgtm

**Pull request merged**